src/components/hero/homeHero.js

```diff
@@ -1,74 +1,67 @@
+    >
       <Box
         sx={{
-          height: '400px',
-          display: 'flex',
-          overflow: 'hidden',
+          maxWidth: "500px",
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "flex-start",
+          gap: "1rem",
         }}
       >
-        <Box sx={{ 
-          flex: '1 1 200px', 
-          padding: '2rem 0', 
-          display: 'flex', 
-          flexDirection: 'column', 
-          justifyContent: 'space-between',
-        }}>
-          <Box>
-            <Typography variant="h1" sx={{
-              color: theme.palette.branding.offWhite,
-              mb: '1rem',
-            }}>
-              {content.title}
-            </Typography>
-            <Typography variant="body2">
-              {content.subheading}
-            </Typography>
-          </Box>
-          <Box>
-            <Button variant='filled' size='medium' component={ Link } to="/features" sx={{
-              color: theme.palette.branding.deepPurple, 
-              textTransform: 'none',
-              backgroundColor: theme.palette.branding.yellow,
-              '&:hover': {
-                backgroundColor: `${ theme.palette.branding.yellow }70`,
-              }
-            }}>
-              <Typography variant='subtitle2'>
-                Why use Helx?
-              </Typography>
-            </Button>
-
-          </Box>
-        </Box>
-        <Box sx={{
-          flex: '2 2',
-          overflow: 'hidden',
-          position: 'relative',
-        }}>
-          <img 
-            src={content.image} 
-            loading="lazy"
-            style={{ 
-              margin: '0 auto', 
-              width: '400px',
-              position: 'absolute',
-              bottom: '-7rem',
-              right: '0'
-            }} 
-          />
-        </Box>
+        <Typography
+          variant="h1"
+          sx={{
+            color: theme.palette.branding.offWhite,
+            margin: 0,
+            lineHeight: 1.3,
+          }}
+        >
+          {content.title}
+        </Typography>
+        <Typography variant="body2">{content.subheading}</Typography>
+        <Button
+          variant="filled"
+          size="medium"
+          sx={{
+            marginTop: "100px",
+            color: theme.palette.branding.deepPurple,
+            textTransform: "none",
+            backgroundColor: theme.palette.branding.yellow,
+          }}
+        >
+          Why use Helx?
+        </Button>
       </Box>
 
-    </MaxWidthWrapper>    
-  )
-}
+      <Box
+        sx={{
+          position: "absolute",
+          width: "40%",
+          top: "4rem",
+          right: 16,
+          [theme.breakpoints.down(BREAKPOINT)]: {
+            top: "revert",
+            bottom: 0,
+            transform: "translate(0, 40%)",
+          },
+        }}
+      >
+        <img src={content.image} loading="lazy" style={{ width: "100%" }} />
```

Comment:
should we remove the lazy loading property for this?

Reply:
Yeah I'll remove it for now. We should probably do some more testing though